### DISCUSSION TITLE
More obvious archive button state

### DISCUSSION THREAD

**Discussion by manuelwehrli** [directus/directus#4816] [Feature Requests]
The button icons to archive and unarchive an item are very similar and difficult to distinguish by a quick look. Can we make something more obvious, maybe by changing the button's color or inserting the archived state into the item's title?

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
